target/riscv/op_helper_cheri.c

```diff
@@ -165,6 +181,10 @@ void HELPER(cspecialrw)(CPUArchState *env, uint32_t cd, uint32_t cs,
                               _host_return_address);
     }
     bool can_access_sysregs = cheri_have_access_sysregs(env);
+    bool is_write = (cs != 0);
```

Comment:
Introducing a variable for this when there are multiple existing, untouched uses of `cs != 0` isn't particularly clean

Reply:
@jrtc27 The same would then apply to `bool can_access_sysregs`. I attempted to use the style of the code around, but I am more than happy to directly use `cs != 0`. I did not spot a variable that would give me the same semantics, but I might have missed it.